src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -2225,4 +2224,4 @@
     </Environment>
   </Provider>
 
-</Providers>
\ No newline at end of file
+</Providers>
```

Comment:
Looks like an unwanted change. Can you please revert it?

Reply:
Oh, new lines are added by the editor to all files. I'll remove it in another editor.. 😅

Reply:
Ah crap, I thought it was the other way around: in this case, please keep the new line!

Reply:
Done and undone the unwanted (wanted) change! 🤣